docs/demos/video-links.md

```diff
@@ -0,0 +1,9 @@
+# Introduction and Demo Videos
+
+There are several on-line videos available to help you quickly understand the concept of Akri and get some quick demos.
+
+1. [Bridge Your IoT Leaf Devices to Local Clusters with Ease Using Akri and Dynamic Resource Allocation](https://www.youtube.com/watch?v=ZZfDne7gMMI) - Latest Akri introduction at KubeCon EU 2024.
+1. [Akri is dope!](https://www.youtube.com/watch?v=KLA3N8xbYPE) - A talk in the Azure Arc Jumpstart channel. Includes a demo of discovering an ONVIF camera with Akri and feeding the stream to a edge AI model.

```

Comment:
To be honest, I don't love the title of this video. If we are going to link to it, can we not add it's title to the documentation?

Reply:
I'm aligned with this, let's just call it "Azure Arc Jumpstart with Akri" or something like that